### PR TITLE
Mk/fifty fifty

### PR DESCRIPTION
**Pull request by mkurvin**

### CODE REVIEW

File: demo/shared/components/fifty-fifty-image.html [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
         </div>
 
-        <div class="col-lg-6">
-            <div class="l-fifty-text right">
-                <h3 class="h-line-mark">Global Biotech Launches New Informative Website</h3>
-                <p>A global BioTech firm partnered with XCentium to develop a patient information website to support a new medicine, which was first in it’s class. </p>
-                <a href="#" class="c-cta btn btn-primary">Read More</a>
+        <div class="col-lg-6 l-fifty__col-2">
+            <div class="l-fifty__content-container container l-fifty-text">
+                <h3 class="l-fifty__heading h-line-mark" contenteditable="true">Global Biotech Launches New Informative Website</h3>
```
[the remark this answers is not in the document]
Comment (justindmyers) [reply demo/shared/components/fifty-fifty-image.html]: Contenteditable still needs to be removed from the markup

File: demo/shared/pages/components/fifty-fifty-image.vue [hunk 2]
```diff
@@ -20,8 +20,16 @@
...
                     imageOverlay: false,
-                    height: false
-                }
+                    height: false,
+                    verticalAlignment: false,
+                    layoutReverse: true
+                },
+                 editableImages: [{
+                    name: 'Background Image',
+                    selector: '.h-mask-img',
```
[the remark this answers is not in the document]
Comment (justindmyers) [reply demo/shared/pages/components/fifty-fifty-image.vue]: This still hasn't been updated

File: src/shared/sass/components/_fifty-fifty-image.scss [hunk 3]
```diff
@@ -33,6 +35,213 @@
...
+        @include make-container-max-widths();
+        margin-left: auto;
+        margin-right: auto;
+    }
+
+    @include xc-component-layout-margin() {
+        margin: $xc-theme-component-margin;
+    }
+
+    @include xc-component-layout-cta() {
```
[the remark this answers is not in the document]
Comment (justindmyers) [reply src/shared/sass/components/_fifty-fifty-image.scss]: xc-component-layout-cta still needs to be removed

**Comment by mkurvin** [comment]
Why does content editable need to be removed? Aren’t we able to edit it in this component?


Matt Kurvin
.: KURVIN Media :.
www.kurvinmedia.com
831 239 9506

File: src/themes/xcentium/sass/components/_fifty-fifty-image.scss [hunk 4]
```diff
@@ -8,22 +8,35 @@
...
-        &.right{
-            .h-line-mark{
-                position: relative;
-                
-                &:before{
-                    position: absolute;
-                    right: 0;
-                    top: -5px;
-                }
+        .h-line-mark{
```
Comment (justindmyers) [inline src/themes/xcentium/sass/components/_fifty-fifty-image.scss]: This is not correct. All the component needs to do is extend the h-line-mark styles like the .c-hero component does. Look at `src\themes\xcentium\sass\components\_hero.scss` for an example.

File: src/themes/xcentium/sass/components/_fifty-fifty-image.scss [hunk 5]
```diff
@@ -8,22 +8,35 @@
 
 .l-fifty{
```
Comment (justindmyers) [inline src/themes/xcentium/sass/components/_fifty-fifty-image.scss]: This still needs to be updated from `.l-fifty` to `.c-fifty`. Please update the HTML and scss accordingly.

**Comment by mkurvin** [comment]
It’s applied through JavaScript. If it’s in the HTML the Sitecore developer will end up copying it into Sitecore meaning it would be editable on the live site.

**Pull request merged**